assets/js/modules/adsense/components/setup/SetupAccountCreate.js

```diff
@@ -97,14 +96,13 @@ export default function SetupAccountCreate() {
 					href="https://support.google.com/adsense/answer/2659101"
 					inherit
 					external
-					dangerouslySetInnerHTML={ sanitizeHTML(
-						__( 'Learn more<span class="screen-reader-text"> about adding a user to an existing AdSense account</span>', 'google-site-kit' ),
-						{
-							ALLOWED_TAGS: [ 'span' ],
-							ALLOWED_ATTR: [ 'class' ],
-						}
-					) }
-				/>
+				>
+					{ __( 'Learn more', 'google-site-kit' ) }
+					{ ' ' }
+					<span className="screen-reader-text">
+						{ __( 'about adding a user to an existing AdSense account', 'google-site-kit' ) }
+					</span>
```

Comment:
See above.

Reply:
Updated to use `createInterpolateElement`